src/messages/en.json

```
@@ -59,5 +59,18 @@
     "Tanya": "Team Lead",
     "Mariya": "Architect",
     "Victor": "Team Driver"
+  },
+  "history": {
+    "titleNotFound": "You haven't executed any requests. Try",
```

nit: просто "try" звучит как-то рвано и неественно, я бы дописала
```suggestion
    "titleNotFound": "You haven't executed any requests. Try sending your first one",
```

---

добавила